siac/hostcmd.go

```diff
@@ -51,6 +52,61 @@ Doing so will override the standard connectivity checks.`,
 	}
 )
 
+// parseSize converts strings of form 10GB to a size in bytes
+// If parsing fails the original value is returned unmodified
+func parseSize(value string) string {
+	var suffix string
+	var size int
```

Comment:
I would consider the variables to be a part of the `if` statement, it's not like they're actually in the way. If you move it and decide that it's too disruptive, you can leave it how it is.